### ISSUE TITLE
cropped bars code while printing generated sheet

### ISSUE THREAD

**Issue by ghost** [Pilooz/techshopio#11]
Hi,

With a 200 bar code sheet, I'm always getting cropped bar codes between pages while trying to print it out on a A4 format
[TechShopIO.pdf](https://github.com/Pilooz/techshopio/files/727185/TechShopIO.pdf)
. It is possible to generate a proper PDF with no crop instead of a plain HTML page ?

**Comment by Pilooz** [comment]
Yes, the adjustable height could cause cropping problem. this is because the page size is not a muliple of all heights of bar code.

This could be cool to add a parameter that fixes the number of barcode line in each page. This parameter could be adjustable ?
